src/Hazelcast.Net/Core/RetryOptions.cs

```diff
@@ -55,9 +53,9 @@ private RetryOptions(RetryOptions other)
         public double Multiplier { get; set; } = 1;
 
         /// <summary>
-        /// Gets or sets the cluster connection timeout in milliseconds.
+        /// Gets or sets the timeout in milliseconds.
         /// </summary>
-        public long ClusterConnectionTimeoutMilliseconds { get; set; } = 20*1000;
+        public long TimeoutMilliseconds { get; set; } = 20*1000;
```

Comment:
Likewise, this is not a generic timeout. It's connection timeout, so I think the old name or `ConnectionTimeoutMilliseconds` is a better choice. 

The default value should be infinite, not 20 sec.

Reply:
ok, I get it, I renamed it because IRetryOptions seemed generic enough that we may want to reuse it in places where it's not about connections... but... never mind, renaming